06week/checkpoint2.js

```diff
@@ -0,0 +1,103 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+let nums = [];
+const strAsNum = strNums.map(str => nums.push(str));
```

Comment:
The map function returns a new array, eliminating the need to push to a new array.  Also, we are transforming the original array from strings to numbers.